MarshalTests/MarshalTests.swift

```diff
@@ -257,8 +268,19 @@ class MarshalTests: XCTestCase {
     }
     
     func testSwiftBasicTypes() {
-        let object: MarshalDictionary = ["int8": NSNumber(value: 100), "int16": NSNumber(value: 32_000), "int32": NSNumber(value: 2_100_000_000), "int64": NSNumber(value: 9_000_000_000_000_000_000), "uint8": NSNumber(value: 200), "uint16": NSNumber(value: 65_000), "uint32": NSNumber(value: 4_200_000_000), "uint64": NSNumber(value: 18_000_000_000_000_000_000), "char": "S"]
-        
+        // broken into two parts because it was timing out during compile time.
+        let partOne: Dictionary = ["int8": NSNumber(value: 100),
+                                   "int16": NSNumber(value: 32_000),
+                                   "int32": NSNumber(value: 2_100_000_000),
+                                   "int64": NSNumber(value: 9_000_000_000_000_000_000)]
+        let partTwo: Dictionary = ["uint8": NSNumber(value: 200),
+                                   "uint16": NSNumber(value: 64_000),
+                                   "uint32": NSNumber(value: 4_200_000_000),
+                                   "uint64": NSNumber(value: 9_000_000_000_000_000_000),
+                                   "char": "S" as AnyObject]
+
+        let object: MarshalDictionary = partOne.reduce(partTwo) { r, e in var r = r; r[e.0] = e.1; return r }
+
```

Comment:
I had to break this into two pieces because it was failing to compile in a timely manner. Also, the uint64 value of '18_000_000_000_000_000_000' was overflowing so I changed it to '9_000_000_000_000_000_000'. I'm not sure how to avoid that or why it worked before and not now.